vizro-core/src/vizro/static/css/fonts/tooltip.css

```diff
@@ -0,0 +1,25 @@
+.tooltip-inner {
+  border-radius: 0;
+  box-shadow: var(--box-shadow-elevation-0);
+  filter: drop-shadow(0 2px 2px #141721);
+  font-size: var(--text-size-01);
+  font-weight: var(--text-weight-light);
+  letter-spacing: var(--letter-spacing-help-text);
+  line-height: var(--text-size-03);
+  max-width: 180px;
+  overflow-wrap: break-word;
+  padding: var(--spacing-01) var(--spacing-02);
+  white-space: pre-wrap;
+}
+
+.bs-tooltip-end .tooltip-arrow::before {
+  right: -3px;
+}
+
+.tooltip.show {
+  opacity: 1;
+}
+
+.tooltip {
+  pointer-events: none;
+}
```

Comment:
@nadijagraca  This is what I added in this [commit](https://github.com/mckinsey/vizro/pull/362/commits/5869127535fc33ff77b1722733fe961d585f40fd) to remove the flickering of the tooltip I've noticed when hovering over the edge.

For context: https://github.com/valor-software/ngx-bootstrap/issues/3075 :)